src/index.css

```
@@ -525,6 +540,19 @@ html {
   }
 }
 
+@media (min-width: 1024px) {
+  .onboarding-shell {
+    --onboarding-shell-max-width: 980px;
+    border-radius: 24px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)